lineman/app/components/group_page/discussions_card/discussions_card.haml

```diff
@@ -12,7 +12,7 @@
       .group-thread-list-empty-access{ng-if: 'howToGainAccess()'}
         %span{translate: 'group_page.thread_list.empty.{{howToGainAccess()}}'}
 
-    .group-thread-list-item{ng-repeat: 'discussion in group.organisationDiscussions() track by discussion.id | orderBy: ["-lastActivityAt"] | limitTo:loaded'}
+    .group-thread-list-item{ng-repeat: 'discussion in group.organisationDiscussions() | orderBy: "-lastActivityAt" track by discussion.id | limitTo:loaded'}
```

Comment:
the position of track by boggles me. Is it also ok at the end of the expression?